src/pages/schaapi.js

```diff
@@ -1,27 +1,102 @@
 import React from "react"
+import Helmet from "react-helmet"
 
+import schaapiLogo from "../images/schaapi-logo.png"
 import sergLogo from "../images/serg.png"
 import tudelftLogo from "../images/tudelft.png"
 
 const SchaapiPage = () => (
-    <div className="text-center">
-        <h1 className="text-5xl">schaapi</h1>
-        <p className="leading-loose text-xl">🐑 Safe Changes for APIs</p>
-
-        <div className="mt-8">
-            <a className="btn btn-green inline-block" href="https://github.com/cafejojo/schaapi">
-                <i className="fa fa-github mr-2" />
-                github.com/cafejojo/schaapi
-            </a>
+    <div className="schaapi-page pt-6">
+        <Helmet title="Schaapi - Café Jojo" />
+
+        <div className="text-center">
+            <img src={schaapiLogo} alt="Schaapi" className="md:w-1/2 px-4" />
+
+            <p className="leading-loose text-xl">🐑 Safe Changes for APIs</p>
+
+            <div className="mt-8">
+                <a className="btn btn-green inline-block" href="https://github.com/cafejojo/schaapi">
+                    <i className="fa fa-github mr-2" />
+                    github.com/cafejojo/schaapi
+                </a>
+            </div>
+
+            <div className="flex justify-center align-bottom items-center mt-8">
+                <a href="http://swerl.tudelft.nl/bin/view/Main/WebHome">
+                    <img src={sergLogo} title="Software Engineering Research Group" height={35} className="mr-4 mt-3" />
+                </a>
+                <a href="https://www.tudelft.nl/">
+                    <img src={tudelftLogo} title="Delft University of Technology" height={50} />
+                </a>
+            </div>
         </div>
 
-        <div className="flex justify-center align-bottom items-center mt-8">
-            <a href="http://swerl.tudelft.nl/bin/view/Main/WebHome">
-                <img src={sergLogo} title="Software Engineering Research Group" height={35} className="mr-4 mt-3" />
-            </a>
-            <a href="https://www.tudelft.nl/">
-                <img src={tudelftLogo} title="Delft University of Technology" height={50} />
-            </a>
+        <div className="mt-16">
+            <div className="content-section bg-delft-blue text-white">
+                <h2>What is a "schaapi"?</h2>
```

Comment:
I don't endorse the term "a schaapi"

Reply:
Then suggest a better title ;)

Reply:
As discussed IRL, changed to "What is Schaapi?"

Reply:
Noice